### PR TITLE
Upgrade plugins + script to upgrade plugins

### PR DESCRIPTION
**Pull request by sylviamoss**

### CODE REVIEW

File: scripts/upgrade_plugins.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#!/bin/zsh
+
+## This script is to be run before a Packer release in order to update
+## all vendored plugins to the latest available release.
+## The SDK is included in the plugin list and will be upgraded as well if a
+## newest version is available.
+## This script should be run in packer's root.
+
+declare -a plugins=(
+    "alicloud"
```
Comment (azr) [inline scripts/upgrade_plugins.sh:10]:
```suggestion
	"alicloud"
```
Just for indentation

File: scripts/upgrade_plugins.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	"vmware"
+	"vsphere"
+	"yandex"
+)
+
+## now loop through the above plugin array
+## update the plugins and the SDK to the latest available version
+for i in "${plugins[@]}"
+do
+   happy=false
```
Comment (azr) [inline scripts/upgrade_plugins.sh:50]: 😄

**Review by azr — APPROVED** [review]
Great work !

File: scripts/upgrade_plugins.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	"ncloud"
+	"openstack"
+	"oracle"
+	"outscale"
+	"parallels"
+	"proxmox"
+	"puppet"
+	"qemu"
+	"scaleway"
+	"sdk"
```
Comment (azr) [inline scripts/upgrade_plugins.sh:35]: nice

**Comment by jboero** [comment]
Super helpful thanks!

**Pull request merged**